ehitchhiking-ui/src/app/shared/components/chat/chat.component.sass

```diff
@@ -0,0 +1,108 @@
+@import "../../../../styles/colors"
+@import "../../../../styles/mixins"
+
+.modal-wrapper
+  .chat
+    overflow-x: hidden
+    .message-zone
+      height: 20rem
+      max-width: 23rem
+      margin: 0.5rem
+      font-size: 0.9rem
+      font-weight: normal
+      display: flex
+      flex-direction: column
+.input-zone
+  display: flex
+  justify-content: space-between
+  .input
+    width: 40rem
+  .send-btn
+    background-color: mat-color($primary, 50)
+    height: 2.8rem
+.modal-back
+  position: absolute
+  top: 0.5rem
+  left: 0.5rem
+  @include flexbox()
+  color: mat-color($primary, 400)
+  &-btn:hover
+    background-color: mat-color($accent, 50)
+  &-btn
+    .btn-text
+      display: inline
+
+
+.message-wrapper
+  display: flex
+  justify-content: flex-end
+
+  .message
+    max-width: 8rem
+    word-wrap: break-word
+    margin-bottom: 0.375rem
+    line-height: 1.5rem
+    position: relative
+    padding: 0.6rem 0.7rem
+    border-radius: 1rem
+    background: $light-gray
+    color: $black
+
+    &:before, &:after
+      content: ""
+      position: absolute
+      bottom: -2px
+      height: 0.625rem
+
+  .person-info
+    display: flex
+    flex-direction: row-reverse
+    .date
+      font-size: small
+      color: $gray
+      height: 1rem
+      position: fixed
+      margin-top: 2rem
+    .image
+      border-radius: 50%
+      width: 2rem
+      height: 2rem
+
+  &.my-message
+    margin-right: 0.5rem
```

Comment:
pls create a local variable for this offset. As far as I understand my-message and :not(.my-message) should have the same offset

Reply:
my-message and :not(.my-message) have a lot of the same sizes but they are reversed, pls create constants for all of them

Reply:
done